modules/microshift-config-yaml.adoc

```diff
@@ -4,81 +4,43 @@
 
 :_content-type: CONCEPT
 [id="microshift-config-yaml_{context}"]
-= Using a YAML configuration file
+= Configuring {product-title}
 
-Configure your {product-title} using a YAML file. The values specified in your configuration file are used when you execute a command.
-// Q: can these values be overriden with CLI flags?
+{product-title} searches for a configuration file in the user-specific directory, `~/.microshift/config.yaml`, then the system-wide `/etc/microshift/config.yaml` directory. Create the configuring file, specifying any settings that should override the defaults, before starting MicroShift.
 
 [id="microshift-yaml-default_{context}"]
 == Default settings
 If you do not create a `config.yaml` file, the default values are used.
-// Q: what creates the default yaml?
```

Comment:
There is a default file in /etc/microshift/config.yaml.example that comes from the RPM package. Nothing reads that, it's just meant as a help to the user. If no config file exists, the defaults are built into the code.

Reply:
Done